source/style-guide/style/code-examples.txt

```diff
@@ -50,6 +35,23 @@ or output examples:
 -  Follow the conventions of the programming language used and preserve
    the capitalization that the author of the code used.
 
+For shell commands, use the following guidelines when creating blocks 
+of code as input or output examples:
+
+-  When showing input, do not include a command prompt (such as $).
+
+-  As often as necessary, show input and output in separate blocks and
+   provide explanations for each. For example, if the input contains
+   arguments or parameters, explain those. If the user should expect
+   something specific in the output, or you want to show only part of
+   lengthy output, provide an explanation.
+
+-  When the command is simple, and there's nothing specific to say
+   about the output, you can show the input and output in the same code
+   block, as users would actually see the code in their own terminal.
+   The inclusion of the command prompt differentiates the input from
+   the output.
```

Comment:
I think it would make sense to suggest using an io-code-block in this case rather than putting the input and output into the same code block. Especially since we're removing the `$` command prompt